.github/workflows/rubygems-deployment.yaml

```diff
@@ -15,4 +15,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+        ruby-version: 3.3
```

Comment:
This doesn't need the patch verion in this? 3.3.6

as per your pr [here](https://github.com/paperkite/paperkite-rubocop/pull/17/files)

Reply:
Nah for deployment it doesn't matter, it'll use latest. It should be detecting it so I probably can remove this line.